commit 5ec2fb163845723f8db407476e190cbeb1d10c46 (refs/changes/58/358/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-09-01T20:28:30+00:00 (3 years, 1 month ago)
    
    FWAPI-246 fwadm(1M) should use the REQ_ID environment variable when present
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/src/fw/lib/util/log.js b/src/fw/lib/util/log.js
index cad4f7fa..f652f59e 100644
--- a/src/fw/lib/util/log.js
+++ b/src/fw/lib/util/log.js
@@ -328,7 +328,7 @@ function createLogger(opts, readOnly) {
 
     log = bunyan.createLogger({
         name: logName,
-        req_id: opts.req_id || mod_uuid.v4(),
+        req_id: opts.req_id || process.env.REQ_ID || mod_uuid.v4(),
         serializers: serializers,
         streams: streams
     });
